pkg/log/syslog/writer.go

```diff
@@ -184,9 +184,12 @@ func (w *writer) writeAndRetry(p Priority, tag, s string) (int, error) {
 	pr := (w.priority & facilityMask) | (p & severityMask)
 
 	if w.conn != nil {
-		if n, err := w.write(pr, tag, s); err == nil {
+		n, err := w.write(pr, tag, s)
+		if err == nil {
 			return n, err
```

Comment:
Nit: `return n, nil` might be more clear here. Not sure what the style guideline is for that. Otherwise LGTM